src/config/config.server.ts

```diff
@@ -231,6 +232,29 @@ export const buildAppConfig = (destConfigPath?: string): AppConfig => {
   // apply build defined production
   appConfig.production = env === 'production';
 
+  const useDynamicLanguageHashes = isNotEmpty(ENV('DYNAMIC_LANGUAGE_HASHES', true)) ? ENV('DYNAMIC_LANGUAGE_HASHES', true) : false;
```

Comment:
>The env var should be DSPACE_DYNAMIC_LANGUAGE_HASHES.

I'm sorry. I got it from this line in the code. 

If I use `DSPACE_DYNAMIC_LANGUAGE_HASHES`, it _does_ work, if I restart the server after a change.

Reply:
Yes, the ```prefix``` logic inside the ENV method was also confusing me first when working on the code.